sys/Makefile

```diff
@@ -17,8 +17,8 @@ ifneq (,$(filter nomac,$(USEMODULE)))
     DIRS += net/link_layer/nomac
 endif
 ifneq (,$(filter transport_layer,$(USEMODULE)))
-	USEMODULE += udp
-	USEMODULE += tcp
+  USEMODULE += udp
+  USEMODULE += tcp
```

Comment:
From my side we can keep the edits if you change it to 2->4 white-spaces (also  in line 48 below), or just revert the changes for the lines.